### PR TITLE
JNG-5526 Refactor service functions to core

### PR DESCRIPTION
**Pull request by robertcsakany**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-5526" title="JNG-5526" target="_blank">TaskJNG-5526</a>  Move Rest / Security / Mapper functions from Services to Core
  </td></table>
  <br />
 

JNG-5526 Refactor service functions to core

### CODE REVIEW

File: judo-runtime-core-jackson/src/main/java/hu/blackbelt/judo/runtime/core/jackson/ObjectMapperFactory.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+public class ObjectMapperFactory {
+
+    public static ObjectMapper createObjectMapper() {
+        SimpleModule customSerializerModule = new SimpleModule()
+                .addSerializer(LocalDateTime.class, new LocalDateTimeSerializer())
+                .addDeserializer(LocalDateTime.class, new LocalDateTimeDeserializer());
+
+        ObjectMapper mapper = new ObjectMapper()
+                .registerModule(new ParameterNamesModule())
+                .registerModule(new Jdk8Module())
```
Comment (noherczeg) [inline judo-runtime-core-jackson/src/main/java/hu/blackbelt/judo/runtime/core/jackson/ObjectMapperFactory.java:24]: JDK8???

File: judo-runtime-core-security-keycloak/src/main/java/hu/blackbelt/judo/runtime/core/security/keycloak/KeycloakRealmSynchronizer.java [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+@Slf4j
+public class KeycloakRealmSynchronizer {
+
+    public enum AccessType {
+        PUBLIC, CONFIDENTIAL, BEARER_ONLY
+    }
+
+    /*
+    @ObjectClassDefinition
```
Comment (noherczeg) [inline judo-runtime-core-security-keycloak/src/main/java/hu/blackbelt/judo/runtime/core/security/keycloak/KeycloakRealmSynchronizer.java:52]: ?

File: judo-runtime-core-security/pom.xml [hunk 3]
```diff
@@ -44,5 +44,13 @@
             <artifactId>hu.blackbelt.judo.meta.asm.model</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>hu.blackbelt.judo</groupId>
+            <artifactId>judo-dao-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>hu.blackbelt.judo</groupId>
+            <artifactId>judo-dispatcher-api</artifactId>
```
Comment (noherczeg) [inline judo-runtime-core-security/pom.xml:53]: Security calling the dispatcher? :D

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: judo-runtime-core-jackson/src/main/java/hu/blackbelt/judo/runtime/core/jackson/ObjectMapperFactory.java (hunk 1 above)
Comment (robertcsakany) [reply judo-runtime-core-jackson/src/main/java/hu/blackbelt/judo/runtime/core/jackson/ObjectMapperFactory.java:24]: Yes this is the way :)

Comment (robertcsakany) [reply judo-runtime-core-jackson/src/main/java/hu/blackbelt/judo/runtime/core/jackson/ObjectMapperFactory.java:24]: Yes, this is the way :)

Comment (bencegelei) [reply judo-runtime-core-jackson/src/main/java/hu/blackbelt/judo/runtime/core/jackson/ObjectMapperFactory.java:24]: 🫨

**Review by gaborflorian — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]

**Review by noherczeg — APPROVED** [review]

**Pull request merged**